mmv1/third_party/terraform/tests/resource_alloydb_cluster_test.go

```diff
@@ -737,3 +737,477 @@ resource "google_kms_crypto_key_iam_binding" "crypto_key2" {
 }
 `, context)
 }
+
+// Validates continuous backups defaults to being enabled with 14d retention, even if not explicitly configured.
+func TestAccAlloydbCluster_continuousBackup_enabledByDefault(t *testing.T) {
```

Comment:
Since AlloyDB clusters take a while to spin up, it would be great to add all of these as steps in an existing test rather than new test cases.

Reply:
AlloyDB clusters are actually really quick to spin up (a couple seconds at most). It's the instances that can take up to 10 minutes. Since we're not doing any instances in this test suite, I think it is a bit cleaner and more legible to keep these tests separate.

That said, I did condense the two update tests into one since that was a pretty trivial change.